content/tutorials/manage-the-rippled-server/installation/install-rippled-on-ubuntu.md

```diff
@@ -31,23 +31,22 @@ Before you install `rippled`, you must meet the [System Requirements](system-req
 
         sudo mkdir /usr/local/share/keyrings/
         wget -q -O - "https://repos.ripple.com/repos/api/gpg/key/public" | gpg --dearmor > ripple-key.gpg
-        sudo mv ripple-key.gpg /usr/local/share/keyrings
+        sudo mv ripple-key.gpg /usr/local/share/keyrings/
 
 
 4. Check the fingerprint of the newly-added key:
 
-        gpg /usr/local/share/keyrings/ripple-key.gpg
+        gpg --import --import-options show-only /usr/local/share/keyrings/ripple-key.gpg
```

Comment:
Good catch on this, this always annoyed me. Seemed wrong, just was always like this.